### ISSUE TITLE
Database?

### ISSUE THREAD

**Issue by jpeg2600** [fifieldt/meshtastic-map-backend#3]
What do you think of using a small database to store nodes informations instead of pickling? I think MongoDB would be great to do just that as you basically push and pull JSON objects, I don't think much code would be needed.

**Comment by fifieldt** [comment]
Hi, yes!

I'd suggest two requirements:
* retain a way to run "lightweight" without having to operate any external daemons (maybe sqlite?)
* make sure it can run on https://railway.app/ - meshtastic project's preferred hosting for these kind of things.

**Comment by jpeg2600** [comment]
I don't know anything of Railway (yay new things to learn!) but it seems like it supports MongoDB : https://docs.railway.app/guides/mongodb 

I understand the concern of an external daemon. I'll experiment a bit with SQLite and see how it goes

**Comment by fifieldt** [comment]
Yeah, railway probably supports everything we need, just added for completeness. If confirming that support gets in the way of your fun, ignore the request and I'll look into it :)
